arex-web-core/src/main/java/com/arextest/web/core/repository/mongo/FSCaseRepositoryImpl.java

```diff
@@ -142,11 +142,11 @@ public List<FSItemDto> queryCases(String workspaceId, String name, List<String>
     }
     if (includeLabels != null && !includeLabels.isEmpty()) {
       query.addCriteria(new Criteria().orOperator(
-          Criteria.where(FSCaseCollection.Fields.labelIds).in(includeLabels)));
+          Criteria.where(FSInterfaceCollection.Fields.labelIds).in(includeLabels)));
```

Comment:
Whether to support the search of interface's label

Reply:
Just adjusting the 'labelIds' field to the base class, the logic has not actually changed.